packages/yoastseo/src/scoring/assessments/seo/ProductSKUAssessment.js

```diff
@@ -65,7 +85,7 @@ export default class ProductSKUAssessment extends Assessment {
 	isApplicable( paper ) {
 		const customData = paper.getCustomData();
 		// Product should either be a simple product with a price, or a product with variants.
```

Comment:
Maybe we don't need this comment anymore?

Reply:
agreed